src/chat/dto/create-chat.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString } from 'class-validator';
+import { ObjectId } from 'mongoose';
+
+export class CreateChatDto {
+  @ApiProperty({ example: 'userId', description: 'User ID' })
+  @IsNotEmpty()
+  @IsString()
+  userId: string | ObjectId;
+
+  @ApiProperty({ example: 'driverId', description: 'Driver ID' })
+  @IsNotEmpty()
+  @IsString()
+  driverId: string | ObjectId;
+}
```

Comment:
You need to add one more field. OrderId or status for this chat. Because I don't know which chat is active.